native/test/test.ml

```diff
@@ -59,21 +59,25 @@ let t_Array () =
     "map2 one element"
     (List.map2 ~f:( + ) [ 1 ] [ 1 ])
     [ 2 ] ;
+
```

Comment:
Odd formatting change. Was it correctly formatted before, or is it correctly formatted now?

Reply:
This was me sorry. `ocamlformat` won't remove excess whitespace, but it doesn't mind the statements being right net to each other.